components/brave_wallet/common/orchard_shard_tree_delegate.cc

```diff
@@ -0,0 +1,55 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/common/orchard_shard_tree_delegate.h"
+
+#include <utility>
+
+namespace brave_wallet {
+
+OrchardTreeState::OrchardTreeState() = default;
```

Comment:
this does not belong in common. Only code used in both browser and renderer process belongs in common

Reply:
Moved to orchard_types